gulpfile.babel.js

```diff
@@ -292,83 +363,115 @@ gulp.task('rpm:linux', ['package:linux'], (done) => {
   });
 });
 
+gulp.task('rpm:linux:32', ['package:linux:32'], (done) => {
+  const redhat = require('electron-installer-redhat');
+
+  redhat(_.extend({}, redhatDefaults, {
+    src: `dist/${packageJSON.productName}-linux-ia32`,
+    arch: 'i386',
+  }), (err) => {
+    console.log('32bit rpm package built');
+    if (err) return done(err);
+    done();
+  });
+});
+
+gulp.task('rpm:linux:64', ['package:linux:64'], (done) => {
+  const redhat = require('electron-installer-redhat');
+
+  redhat(_.extend({}, redhatDefaults, {
+    src: `dist/${packageJSON.productName}-linux-x64`,
+    arch: 'amd64',
+  }), (err) => {
+    console.log('32bit rpm package built');
+    if (err) return done(err);
+    done();
+  });
+});
+
 gulp.task('make:linux', ['deb:linux', 'rpm:linux'], (done) => {
```

Comment:
Since run sequence runs inside of a task, it doesn't properly merge dependencies and therefore I was getting rebuilds in the 4-8 times just by running `make:linux`.